k8s/test-validation.py

```
@@ -0,0 +1,318 @@
+            'rbac.authorization.k8s.io/v1',
+            'policy/v1beta1',
+            'autoscaling/v2',
+            'monitoring.coreos.com/v1'
+        }
+
+    def validate_yaml_syntax(self, file_path: Path) -> bool:
+        """Validate YAML syntax"""
+        try:
+            with open(file_path, 'r', encoding='utf-8') as f:
+                yaml.safe_load_all(f)
+            return True
```

Ruff (0.12.2)

Consider moving this statement to an `else` block

(TRY300)